docs/getting-started/integrations-cdn/vuejs-v2.md

```diff
@@ -0,0 +1,147 @@
+
+Then, include the CKEditor&nbsp;5 scripts and styles. All necessary scripts and links are shown in the HTML snippet below. You can copy and paste them into your `index.html` file. Open-source and premium features are in separate files, so there are different tags for both types of plugins. Add tags for premium features only if you use them.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" href="/favicon.ico" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		
+		<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.css" />
+		<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5.umd.js"></script>
+		<!-- Add if you use premium features -->
+		<link rel="stylesheet" href="https://cdn.ckeditor.com/ckeditor5-premium-features/{@var ckeditor5-version}/ckeditor5-premium-features.css" />
+		<script src="https://cdn.ckeditor.com/ckeditor5/{@var ckeditor5-version}/ckeditor5-premium-features.umd.js"></script>
+		<!--  -->
+
+		<title>CKEditor 5 - Vue 2</title>
+	</head>
+	<body>
+		<div id="app"></div>
+		<script type="module" src="/src/main.js"></script>
+	</body>
+</html>
+
+```
+
+To create an editor instance, you must first import the editor and the component modules into the root file of your application (for example, `main.js` when generated by create-vue).
+
+```js
+import Vue from 'vue';
+import CKEditor from '@ckeditor/ckeditor5-vue2';
+import App from './App.vue';
+
+Vue.use( CKEditor );
+
+new Vue( { render: ( h ) => h( App ) } ).$mount( '#app' );
+```
+
+Use the `<ckeditor>` component inside the template tag. The below example shows how to use the component with open-source plugins.
+
+```html
+<template>
+	<div id="app">
+		<ckeditor :editor="editor" v-model="editorData" :config="editorConfig"></ckeditor>
+	</div>
+</template>
+
+<script>
+const {
+	ClassicEditor,
+	Essentials,
+	Bold,
+	Italic,
+	Paragraph
+} = CKEDITOR;
+
+export default {
+	name: 'app',
+	data() {
+		return {
+			editor: ClassicEditor,
+			editorData: '<p>Hello from CKEditor 5 in Vue 2!</p>',
+			editorConfig: {
+				licenseKey: 'GPL',
+				plugins: [ Bold, Essentials, Italic, Paragraph ],
+				toolbar: [ 'undo', 'redo', '|', 'bold', 'italic' ]
+			}
+		};
+	}
+};
+</script>
+```
+
+The following example shows how to use the component with premium plugins. To make it work, you need to provide a proper license key.
+
+```html
+<template>
+	<div id="app">
+		<ckeditor :editor="editor" v-model="editorData" :config="editorConfig"></ckeditor>
+	</div>
+</template>
+
+<script>
+const {
+	ClassicEditor,
+	Essentials,
+	Bold,
+	Italic,
+	Paragraph,
+	Mention
+} = CKEDITOR;
+const { SlashCommand } = CKEDITOR_PREMIUM_FEATURES;
+
+export default {
+	name: 'app',
+	data() {
+		return {
+			editor: ClassicEditor,
+			editorData: '<p>Hello from CKEditor 5 in Vue 2!</p>',
+			editorConfig: {
+				licenseKey: '<YOUR_LICENSE_KEY>',
```

Comment:
I deliberately didn't add "Or GPL" here. I divided the snippet into two - for open-source and premium plugins. So, the license key is the only option here.

Reply:
Alright, sorry, I missed that part.

Reply:
No problem 👍